core/src/main/java/org/apache/iceberg/BaseReplacePartitions.java

```diff
@@ -25,8 +25,8 @@
 
 public class BaseReplacePartitions
     extends MergingSnapshotProducer<ReplacePartitions> implements ReplacePartitions {
-  BaseReplacePartitions(TableOperations ops) {
-    super(ops);
+  BaseReplacePartitions(String tableName, TableOperations ops) {
+    super(tableName, ops);
```

Comment:
Nit: Other places use `name`.

Reply:
I felt tableName was more appropriate. I have changed it as such in the code I added/modified. 
There are still uses of name though, e.g in Transaction api. I haven't changed them.  Let me know if u feel strong in using `name` or changing to `tableName` everywhere. I'm fine with either